packages/babel-preset-react-app/create.js

```diff
@@ -7,6 +7,13 @@
 'use strict';
 
 const path = require('path');
+const fs = require('fs');
+
+const appDirectory = fs.realpathSync(process.cwd());
+const resolveApp = relativePath => path.resolve(appDirectory, relativePath);
+
+const hasFlowConfig = fs.existsSync(resolveApp('.flowconfig'));
+const hasTSConfig = fs.existsSync(resolveApp('tsconfig.json'));
```

Comment:
Please remove this autodetection magic.

Reply:
By importing a `.ts` or `.tsx` file I imagine? We don't need to turn this on at the preset, webpack can tell it to be on when the extension is matched.

Reply:
@Timer I mean that on this file we can't turn on both flow and typescript presets at the same time, they conflicted with each other; at least it happened when I tried on previous versions, I can check again.

Reply:
If we register a different webpack loader for TS than JS this shouldn't be an issue.

Alternatively, we can make our preset behave the same way the TS preset does and check file extension for what to enable.